### PR TITLE
add server timing directive

### PR DESCRIPTION
**Pull request by damaneice**
Part 1 of  https://github.com/github/voltron/issues/110

### CODE REVIEW

File: response_builder.go [hunk 1]
```diff
@@ -63,6 +64,11 @@ func (rb *responseBuilder) SetFragments(results []*multiplexer.Result) {
 	}
 }
 
+func (rb *responseBuilder) SetDuration(duration int64) {
+	outputHtml := bytes.Replace(rb.body, []byte("<view-proxy-timing></view-proxy-timing>"), []byte(strconv.FormatInt(duration, 10)), 1)
```
Comment (BlakeWilliams) [inline response_builder.go:68]:
```suggestion
	outputHtml := bytes.Replace(rb.body, []byte("<view-proxy-timing></view-proxy-timing>"), []byte(strconv.Itoa(duration)), 1)
```

Minor, but I think `Itoa` is a shortcut for `FormatInt(int64, int)`. Thoughts?

Comment (damaneice) [reply response_builder.go:68]: I couldn't get `Itoa` to work with `int64` But it does work with `int`.

File: server.go [hunk 2]
```diff
@@ -197,6 +197,7 @@ func (s *Server) CreateHandler() http.Handler {
 }
 
 func (s *Server) handleRequest(w http.ResponseWriter, r *http.Request, route *Route, parameters map[string]string, ctx context.Context) {
+	start_time := time.Now()
```
Comment (BlakeWilliams) [inline server.go:200]:
```suggestion
	startTime := time.Now()
```

File: server.go [hunk 3]
```diff
@@ -235,6 +236,8 @@ func (s *Server) handleRequest(w http.ResponseWriter, r *http.Request, route *Ro
 	resBuilder.SetLayout(results[0])
 	resBuilder.SetHeaders(results[0].HeadersWithoutProxyHeaders(), results)
 	resBuilder.SetFragments(results[1:])
+	elapsed := time.Since(start_time)
```
Comment (BlakeWilliams) [inline server.go:239]:
```suggestion
	elapsed := time.Since(startTime)
```

Minor, but Go conventionally uses camel case for variable names

**Review by BlakeWilliams — APPROVED** [review]
I had one small comment about variable casing, but otherwise LGTM

**Pull request merged**